controller/db/external_jwt_signer_store.go

```diff
@@ -43,8 +43,11 @@ const (
 	FieldExternalJwtSignerAudience        = "audience"
 	FieldExternalJwtSignerClientId        = "clientId"
 	FieldExternalJwtSignerScopes          = "scopes"
+	FieldExternalJwtSignerTargetToken     = "targetToken"
 
 	DefaultClaimsProperty = "sub"
+
+	TargetTokenAccess = "ACCESS"
```

Comment:
No TargetTokenID ?

Reply:
Not needed at this level. This is the storage level and can't access values up the dep chain (i.e. in the rest definitions). This is the only default value necessary here.